.travis.yml

```diff
@@ -39,15 +43,18 @@ install:
   - python setup.py sdist && version=$(python setup.py --version) && pushd dist  && pip install WOFpy-${version}.tar.gz && popd
 
 before_script:
-# Commented out due to external service not working. 8/30/17. lsetiawan.
-#  - python wof/examples/flask/cbi/build_cbi_cache.py
   - python wof/examples/flask/swis/runserver_swis.py --config=./test/test_swis_config2.cfg --connection=sqlite:///./test/test_swis2.db &
```

Comment:
@lsetiawan this made me realize that all the `cbi` tests are skipped anyways :smile:

See https://travis-ci.org/ocefpaf/WOFpy/jobs/270205139#L915

Is that something we should look into?

Reply:
@ocefpaf Nah. Not for this release. And it's not really regarding ODM2. Someone else can pick it up 😜

Reply:
Cool. So I'll make another commit to put all `cbi` related tests in the new scenario. That way we get less noise on the `default` scenario. Hold on...